universal-application-tool-0.0.1/app/views/admin/questions/QuestionConfig.java

```diff
@@ -111,12 +111,14 @@ private QuestionConfig addAddressQuestionConfig(AddressQuestionForm addressQuest
             .setId("address-question-disallow-po-box-checkbox")
             .setFieldName("disallowPoBox")
             .setLabelText("Disallow post office boxes")
+            .setValue("true")
             .setChecked(addressQuestionForm.getDisallowPoBox())
             .getContainer(),
         FieldWithLabel.checkbox()
             .setId("address-question-include-none-checkbox")
             .setFieldName("noAddress")
             .setLabelText("Include \"No address\" option")
+            .setValue("true")
```

Comment:
Should we add this now or wait till we have implemented the "no address" option?

Reply:
I think this should be updated now, whether or not we have implemented the option. Alternatively, we could get rid of the checkbox entirely.

What do you think?

Reply:
I deleted the checkbox option entirely.